### ISSUE TITLE
Mobile images for Color page

### ISSUE THREAD

**Issue by mjabbink** [carbon-design-system/carbon-website#154]
the issues we would create for this page still apply because they are the same for Carbon.

For mobile there are specific mobile friendly images. The “Digital accessibility” section for example should be like current DL

https://www.ibm.com/design/language/elements/color-ui

Expected:

![33766D9F-5FDF-4096-84A7-B8EFD268C3DB](https://user-images.githubusercontent.com/32881239/62831491-e014d780-bc3d-11e9-8112-8936bdec6a02.jpeg)

Staging:

![4D9FF76E-3E16-4E4F-BA1B-5C34478AF0B4](https://user-images.githubusercontent.com/32881239/62831494-f3c03e00-bc3d-11e9-9c58-26e88483d86c.jpeg)

**Comment by alisonjoseph** [comment]
transferring this to the Carbon website repo so we can make image updates there
